test/ampycloud/test_layer.py

```diff
@@ -127,3 +129,15 @@ def test_unstable_layers():
                                       delta_mul_gain=0.95, mode='delta')
 
     assert best_ncomp == 1
+
+
+def test_identical_pts():
```

Comment:
here we could add a test to check what happens if all points are at the same height but one

Reply:
Done.